src/context/ThemeContext.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+
+export const themes = {
+    light: {
+        forground: '#242526',
+        background: '#fff',
+        backgroundNav: '#555'
+    },
+    dark: {
+        forground: '#fff',
+        background: '#242526',
+        backgroundNav: '#242424'
+    }, 
+};
+
+//tell context what properties to expect 
+export const ThemeContext = React.createContext({
+    themes: {},
+    toggleTheme: () => {}
+});
+
+
+export const ThemeProvider = (props) => {
+    //state
+    const [theme, setTheme] = useState(themes.light);
+
+    const toggleTheme = () => {
+        setTheme((previousThemeValue)=> 
+          previousThemeValue === themes.dark ? themes.light : themes.dark
+        );
+      };
+
```

Comment:
Technically you are not setting the background color in your theme, you're just applying the dark background color to all of your components instead. A way to check this is when you overscroll in dark mode, you'll still see some white. To fix this, you'd want to do this in your provider.

``` 
 useEffect(() => {
    document.body.style.background = theme.background;
  }, [theme]);
```